pkg/storage/storage.go

```diff
@@ -1,39 +1,150 @@
 package storage
 
 import (
+	"fmt"
+	appsv1 "github.com/openshift/api/apps/v1"
 	"github.com/pkg/errors"
+	"github.com/redhat-developer/ocdev/pkg/component"
 	"github.com/redhat-developer/ocdev/pkg/occlient"
+	"github.com/redhat-developer/ocdev/pkg/util"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
 )
 
-func Add(client *occlient.Client, config *occlient.VolumeConfig) (string, error) {
-	output, err := client.SetVolumes(config,
-		&occlient.VolumeOpertaions{
-			Add: true,
-		})
+const StorageLabel = "app.kubernetes.io/storage-name"
+
+type StorageInfo struct {
+	Name string
+	Size string
+	Path string
+}
+
+// Add adds storage to given component of given application
+func Add(client *occlient.Client, name string, size string, path string, applicationName string, componentName string) (string, error) {
```

Comment:
in all other function, we have opposite order for `applicationName` and `componentName`. `componentName` is before `applicationName` it would be good to keep it consistent.

Reply:
fixed